hypothesis-python/tests/cover/test_observability.py

```diff
@@ -69,6 +69,42 @@ def test_observability():
             )
 
 
+def test_capture_unnamed_arguments():
+    @given(st.integers(), st.floats(), st.data())
+    def f(v1, v2, data):
+        data.draw(st.booleans())
+
+    with capture_observations() as observations:
+        f()
+
+    test_cases = [tc for tc in observations if tc["type"] == "test_case"]
+    for test_case in test_cases:
+        assert list(test_case["arguments"].keys()) == [
+            "generate:v1",
+            "generate:v2",
+            "generate:data",
+            "generate:Draw 1",
+        ], test_case
+
+
+def test_capture_named_arguments():
+    @given(named1=st.integers(), named2=st.floats(), data=st.data())
+    def f(named1, named2, data):
+        data.draw(st.booleans())
+
+    with capture_observations() as observations:
+        f()
+
+    test_cases = [tc for tc in observations if tc["type"] == "test_case"]
+    for test_case in test_cases:
+        assert list(test_case["arguments"].keys()) == [
+            "generate:named1",
+            "generate:named2",
+            "generate:data",
+            "generate:Draw 1",
```

Comment:
Previously `data.draw` was reported as `Draw 1: `, noting both the absence of `generate:` and the trailing `: `. I changed it for consistency with the other reports but this is possibly undesired. 

I'm also not sure when `test_case["arguments"]` would be anything except `generate:`, so possibly we should strip that?

Reply:
Stripping makes sense to me 👍